c-pallets/segment-book/src/lib.rs

```diff
@@ -96,13 +96,16 @@ pub mod sr25519 {
 	}
 
 	sp_runtime::app_crypto::with_pair! {
+		// I am online? seems the comment was copied pasted from Substrate
 		/// An i'm online keypair using sr25519 as its crypto.
 		pub type AuthorityPair = app_sr25519::Pair;
 	}
 
+	// I am online? seems the comment was copied pasted from Substrate
 	/// An i'm online signature using sr25519 as its crypto.
 	pub type AuthoritySignature = app_sr25519::Signature;
 
```

Comment:
Yes, the way I use the offline working machine is to learn im online. I have removed these comments in the newly modified version:
https://github.com/CESSProject/cess/pull/88/files#diff-e315e38811b02c01aead9143bcfe8a7d0db178f24a154bad1431a08648393296L103